tests/cephadm/test_cephadm_operations_via_admin.py

```diff
@@ -0,0 +1,43 @@
+import os
+
+from cli.cephadm.cephadm import CephAdm
+
+
+class CephadmOperationsAdmin(Exception):
+    pass
+
+
+def run(ceph_cluster, **kw):
+    """Verify cephadm operations via admin node for managing and expanding cluster
+    Args:
+        **kw: Key/value pairs of configuration information to be used in the test.
+        e.g
+        test:
+            name: test_cephadm_operations_via_admin_node
+            desc: Verify cephadm operations via admin node for managing and expanding cluster
+            config:
+                label: _admin
+            polarion-id: CEPH-83573725
+            module: test_cephadm_operations_via_admin.py
+    """
+    config = kw.get("config")
+    label = config.get("label")
+    files = config.get("files")
+    installer = ceph_cluster.get_ceph_object("installer")
+    # Fetching node without _admin label
+    node = [
+        node for node in ceph_cluster.get_nodes() if "_admin" not in node.role.role_list
+    ][0]
+    hostname = node.hostname
+    # Adding _Admin label in node
+    exp_out = f"Added label _admin to host {hostname}"
+    result = CephAdm(installer).ceph.orch.label.add(hostname, label)
+    if result != exp_out:
+        raise CephadmOperationsAdmin("Fail to add label")
+    # Verify files in new admin node
+    result = [os.path.exists(file) for file in files]
+    if False in result:
+        raise CephadmOperationsAdmin(f"Files {files} not present")
+    # Operation using orch command in new admin node
+    CephAdm(node).ceph.orch.ls()
```

Comment:
Just checking, shouldn't we be checking the output of the above command? I mean should we check the out to verify if the execution via the admin node succeeded or not? 

Just a check, please ignore if I got it wrong

Reply:
Thanks @pranavprakash20 

As per the test case we need to perform any orch command for verification.
For simplicity in current test case we are using ls command.
https://github.com/red-hat-storage/cephci/blob/master/cli/ceph/orch/orch.py#L26

Yes,It is good to verify same.
Since "ceph orch ls" command have multi-line line output, So verifying it with one column name "RUNNING" from output.